### ISSUE TITLE
rhbz1928860.at fails with NM versions without OVS support

### ISSUE THREAD

**Issue by mbiebl** [firewalld/firewalld#935]
In Debian, NetworkManager is built without OpenVSwitch support.
Which means `src/tests/integration/rhbz1928860.at`  has no chance to succeed.
Atm this means patching out this test : https://salsa.debian.org/utopia-team/firewalld/-/blob/debian/master/debian/patches/Disable-integration-rhbz1928860.at-test.patch

Ideally, I'd like to drop this patch an firewalld skip `rhbz1928860.at` if the ovs plugin is missing.
Asking on #nm, thaller suggested the following:

```
busctl get-property org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.NetworkManager Capabilities

if the array contains "2", it means NM_CAPABILITY_OVS
```
Running this on Debian, I get
```
busctl get-property org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.NetworkManager Capabilities
au 1 1
```

**Comment by mbiebl** [comment]
@thom311 ^

**Comment by mbiebl** [comment]
Or an alternative as suggested by @thom311
```
$ busctl -j get-property org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.NetworkManager Capabilities | jq -e '.["data"] | index(2)' >/dev/null
$ echo $?
1
```

**Comment by thom311** [comment]
or maybe 
```shell
busctl get-property \
       org.freedesktop.NetworkManager \
       /org/freedesktop/NetworkManager \
       org.freedesktop.NetworkManager \
       Capabilities | \
    grep -q '^au [0-9]\+ .*\<2\>'
```
which has no `jq` dependency.

**Comment by erig0** [comment]
Thanks. We can use the grep version to check for the support. The test will be skipped if it's not available.

I'll try to get to it this week.

**Comment by erig0** [comment]
I think this is fixed in #938, but I want to enable integration testsuite in CI as well.

Integration tests fail in CI (#938), but in my local VM it works fine. It looks like the `--change-interface` request is not being forwarded to NetworkManager. However, prior to that (and firewalld reload) we see `nmcli conn up dummy0` causes NM to place dummy0 into the default zone. So at least that part works. Difficult to debug since I can't reproduce it locally.

**Comment by erig0** [comment]
Fixed via #938. Thanks for the assist @mbiebl and @thom311.
